src/components/dialogs/inputs/input-hooks.js

```diff
@@ -864,3 +736,81 @@ export const useCSVReader = ({ label, header }) => {
     }, [selectedFile, intl, header]);
     return [selectedFile, setSelectedFile, field, fileError];
 };
+
+export const useRadioValue = ({
+    label,
+    possibleValues = [],
+    defaultValue,
+    id,
+    validation = {},
+    inputForm,
+    doTranslation = true,
```

Comment:
Is there a use case when we want to avoid translation ?

Reply:
Not yet, but in the future, if we re-use this component for something like "Select your age", we will not have to include every numbers in the en.json and fr.json files.